tests/test_task.py

```diff
@@ -433,6 +437,19 @@ def test_keep_non_empty_dirs(self, tmpdir):
             assert expected == os.path.exists(filename)
         assert os.path.exists(tmpdir['dir'])
 
+    def test_clean_in_reverse_lexical_order(self, tmpdir):
```

Comment:
The test name is misleading because it actually does not ensure reverse lexical order.

Reply:
I agree. I'm having a bit of a tough time finding a proper name though. An option I considered is to actually do ensure that the order is that one, by patching both `os.rmdir` and `os.remove`, what would you think about this?